com.unity.render-pipelines.universal/Shaders/Utils/TileDeferred.shader

```diff
@@ -329,9 +339,14 @@ Shader "Hidden/Universal Render Pipeline/TileDeferred"
             #pragma exclude_renderers d3d11_9x gles gles3 glcore
             #pragma target 4.5
 
+            #pragma multi_compile_fragment _ _DEFERRED_ADDITIONAL_LIGHT_SHADOWS
+            #pragma multi_compile_fragment _ _SHADOWS_SOFT
+            #pragma multi_compile_fragment _ LIGHTMAP_SHADOW_MIXING
+            #pragma multi_compile_fragment _ SHADOWS_SHADOWMASK
+
             #pragma multi_compile_fragment _LIT
             #pragma multi_compile_fragment _ _GBUFFER_NORMALS_OCT
-            #pragma multi_compile_fragment _ _DEFERRED_SUBTRACTIVE_LIGHTING
```

Comment:
This could be a breaking change for custom shaders

Reply:
I renamed `_DEFERRED_SUBTRACTIVE_LIGHTING` into `_DEFERRED_MIXED_LIGHTING` This #define is only used by the deferred shaders (stendilDeferred.shader and TileDeferred.shader), and since there has been no official release yet, it should not break anybody.